site_scons/buildutils.py

```diff
@@ -34,6 +36,373 @@
     TPathLike = TypeVar("TPathLike", Path, str)
 
 
+class Option:
+    """Object corresponding to SCons configuration option"""
+
+    def __init__(self,
+            name: str,
+            description: str,
+            default: Union[bool, str],
```

Comment:
To my comment above, this would become:
```suggestion
            default: "Union[bool, str]",
```